.eslintrc.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Good work creating the app with CRA! 💯 

- [ ] Kindly check this error in the console when try to run the command `npm run start`

![image](https://github.com/techmoves/math-magicians/assets/93630700/ce80122d-f02a-4a44-b126-d9e2548d8d8f)